index.html

```diff
@@ -10,228 +10,307 @@
+            <div class="grid-container-desktop order">
+                <div class="title">
+                    <h1 class="mobile-title1">Multi-Post<br>Stories</h1>
+                    <h1 class="desktop-title2">Uber Navigation</h1>
                 </div>
-            </li>   
-            <!-- Fourth Card -->
-            <li class="card-container4">
-                <img src="./images/Snapshoot4.png" class="card-container-img4" alt="Snapshoot4">
+                <div class="title experience">
 
-                <h2 class="title-project4">Multi-Post <br> Stories</h2>
+                    <ul class="flex-container-list1">
+                        <li>
+                            <h2 class="mobile-version-h2">CANOPY</h2>
+                            <h2 class="desktop-version-h22">Uber</h2>
+                        </li>
+                        <li class="mobile-version-li">Back End Dev</li>
+                        <li class="desktop-version-li1">Lead Developer</li>
+                        <li class="mobile-version-li2">2015</li>
+                        <li class="desktop-version-li2">2018</li>
+                    </ul>
+                </div>
+                <div class="paragraph">
+                    <p class="mobile-para2">A daily selection of privately<br> personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <p class="desktop-para">A smart assistant to make driving more safe, efficient, and fun by unlocking
+                        your most expensive computer: your car.</p>
 
-                <div class="card-content4">
-                    <ul>
-                        <li class="type4">
-                            <p class="t1">CANOPY</p>
-                            <p class="t2"><img src="./images/Circle.png"/> Back End Dev</p>
-                            <p class="t3"> <img src="./images/Circle.png"/> 2015</p>
-                        </li>  
-                    </ul> 
-
-                    <p class="text4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-
-                    <ul class="tech4">
-                        <li class="language1">html</li>
-                        <li class="language2"> css</li>
-                        <li class="language3">javascript</li>
+                </div>
+                <div class="languages1">
+                    <ul class="flex-lang">
+                        <li><span>html</span></li>
+                        <li><span class="desktop-version-span-container">Ruby on rails</span></li>
+                        <li><span>css</span></li>
+                        <li><span>javaScript</span></li>
+                    </ul>
+                </div>
+                <div class="btn-see-projects1">
+                    <a href="#" class="button">See Project</a>
+                </div>
+            </div>
+        </section>
+        <!-- Section 2 : About Myself -->
```

Comment:
I am not able to see the whole design of About Section.
Here is the comparison of the designs:

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221285105-489a8f5e-b6e3-43ff-944e-faa847a91de1.png)| ![image](https://user-images.githubusercontent.com/61361037/221285173-0c03d2fb-4dd1-4e96-b381-c843a1fa7bd1.png)|

1. I will recommend to use `width:100%` or `less` so that the sections width may fit of various screen sizes for desktop versions.
2.

Reply:
Well done on making some changes here 🎉 however some issues are still present, the main one being an overflow issue where I have to scroll horizontally to see the whole content. It happens when I look at page at the full-width of my computer, it works fine when I inspect. I recorded a video explaining that [here](https://www.loom.com/share/b5e01e79630b49c8a8cb77852071b55d) 🔗